traffic_monitor/build/traffic_monitor.init

```diff
@@ -63,7 +63,7 @@ start() {
 
         # Start daemons.
         echo -n $"Starting $name: "
-        daemon nohup $prog $options < /dev/null > /opt/traffic_monitor/var/log/traffic_monitor.log 2>&1 &
+        daemon nohup $prog $options < /dev/null >> /opt/traffic_monitor/var/log/traffic_monitor.log 2>&1 &
```

Comment:
Nice find. I guess this is just capturing anything that TM `fmt.Println`s to stdout? Which if TM is doing that, it would be a bug. So maybe we just shouldn't output to `/opt/traffic_monitor/var/log/traffic_monitor.log` at all?

Reply:
Good point, and anything that the command `daemon` or `nohup` outputs to stdout is captured by journald anyway. Removed in 3914e82170